Dockerfile

```diff
@@ -6,16 +6,27 @@ FROM codait/max-base:v1.1.3
 
 WORKDIR /workspace
 
-RUN wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} && \
-  tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file}
+ARG use_pre_trained_model=true
+
+RUN if [ "$use_pre_trained_model" = "true" ] ; then\
+    # download pre-trained model artifacts from Cloud Object Storage
+    wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} &&\
+            tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file} ; fi
```

Comment:
This is likely a stylish issue: You may want to add some indentation and move down `fi` here.